.github/workflows/swizzle-test.yml

```
@@ -0,0 +1,58 @@
+name: Swizzle Test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['12']
+    steps:
+      - uses: actions/checkout@v1
```

```suggestion
      - uses: actions/checkout@v2
```

`actions/checkout` has a new stable release at `v2`: https://github.com/actions/checkout/releases/tag/v2.0.0
Latests stable is `v2.3.4`, but the `v2` tag points to latest `v2` release as well.